utils/get_supported_scylla_base_versions.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+import os
+
+from pkg_resources import parse_version
+from sdcm.utils.version_utils import is_enterprise
+from sdcm.utils.common import get_s3_scylla_repos_mapping
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+
+# We support to migrate from specific OOS version to enterprise
+supported_src_oos = {'2021.1': '4.3', '2020.1': '4.0', '2019.1': '3.0'}
+# If new support distro shared repo with others, we need to assign the start support versions. eg: centos8
+start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'}}
+
+
+class UpgradeBaseVersion:  # pylint: disable=too-many-instance-attributes
+
+    oos_start_support_version = None
```

Comment:
Typo, must be OSS -> Open Source Scylla.

Reply:
Fixed all same typo, thanks